pkg/cmd/auth/shared/login_flow_test.go

```diff
@@ -179,25 +181,27 @@ func TestLogin(t *testing.T) {
 					// simulate that the public key file has been generated
 					_ = os.WriteFile(keyFile+".pub", []byte("PUBKEY asdf"), 0600)
 				})
-				opts.sshContext = ssh.Context{
-					ConfigDir: dir,
-					KeygenExe: "ssh-keygen",
-				}
+				opts.sshContext = ssh.NewContextForTests(dir, "ssh-keygen")
 			},
 			wantsConfig: map[string]string{
 				"example.com:user":         "monalisa",
 				"example.com:oauth_token":  "ATOKEN",
 				"example.com:git_protocol": "ssh",
 			},
 			stderrAssert: func(t *testing.T, opts *LoginOptions, stderr string) {
+				sshDir, err := opts.sshContext.SshDir()
+				if err != nil {
+					t.Errorf("Could not load ssh config dir: %v", err)
+				}
```

Comment:
Not something that has to be fixed, but I wonder if code like this could be simplified as:

```suggestion
				sshDir, err := opts.sshContext.SshDir()
				require.NoErrorf(t, err, "Could not load ssh config dir")
```